app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "How to choose your teacher training course"
+subcategory: Postgraduate teacher training
+description: |-
+  Find out how to choose your postgraduate teacher training course based on your eligibility, the qualifications it awards, and the course provider.
+related_content:
+    Choosing the right teacher training course provider : "/blog/choosing-the-right-teacher-training-course-provider" 
+    Train to teach if you have a degree : "/train-to-be-a-teacher/if-you-have-a-degree"
+    Your teacher training application : "/how-to-apply-for-teacher-training/teacher-training-application"
+external_content:
+    - content/train-to-be-a-teacher/promos/find-your-course
+promo_content:
+    - content/train-to-be-a-teacher/promos/adviser-promo-degree
+navigation: 20.25
+navigation_title: How to choose your course
+navigation_description: Find out how to choose your postgraduate teacher training course.
+keywords:
+  - becoming a teacher
+  - teacher training
+  - pgce
+  - qts
+  - Teach First
+  - Postgraduate teaching apprenticeship
+
+---
+
+All postgraduate teacher training courses include time spent in school placements with some theoretical learning. 
+
+The main differences between courses are: 
+
+* what you need to get onto the course 
```

Comment:
Can we be slightly clearer here? Are we talking about qualifications?

Reply:
I think we're mainly talking about qualifications and visas, but it might be worth keeping it broad in case there are other eligibility requirements? So we say:

> It's important to check the requirements for each course you apply to, so you do not waste time on an application you’re unlikely to be successful with. 

and then specifically talk about quals and visas, but this could also include things like experience needed for a salaried training course (not sure if there are ever any other eligibility requirements providers can specify @MaeveRoseveare @Kateharries?)

Reply:
Yeah, it can also be things like proving ability to speak English to a good level too. Some people also can start their DBS checks earlier.